src/styles/components/BlockTitleDescription.scss

```diff
@@ -0,0 +1,32 @@
+.BlockTitleDescription {
+  &:nth-of-type(even) {
+    background-color: color('light-grey');
+  }
+
+  &__container {
+    max-width: 35rem;
+    
+    @include media('lg-up') {
+      //.block-width
+      max-width: 74rem;
```

Comment:
Hmm, there might be a good way to leverage Sass to DRY this up.
Something like...
```
$container-md: 35rem; (or $block-width-md or w/e)
$container-lg: 74rem;  (or $block-width-lg or w/e)
```
& then

```
max-width: $container-md;
@include media('lg-up') {
    max-width: $container-lg;
}
```